.github/workflows/build.yml

```diff
@@ -137,6 +137,17 @@ jobs:
         run: python .github/scripts/rat.py
         env:
           GITHUB_WORKSPACE: ${{ github.workspace }}
+      - name: Check generated documentation
+        if: always()
```

Comment:
Hm, `always()` means this will run even if a previous step failed. Maybe we should remove this so the step is not run in the event that the "Compile and validate" step fails.